src/main/java/uk/gov/hmcts/reform/idam/client/models/test/UserGroup.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.reform.idam.client.models.test;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class UserGroup {
```

Comment:
Do you really need this class. Can you not just have `private final String userGroup` in `CreateUserRequest` ?